### DISCUSSION TITLE
窗口最小化到任务栏

### DISCUSSION THREAD

**Discussion by KXST6666** [Meloong-Git/PCL#772] [提问] [answered]
点击PCL软件右上角的“X”总是会退出软件，我希望可以有选择性的然后让它最小化到任务栏里面，需要的时候可以点出来，不知道实现这个功能会不会太难，我作为小白不太懂这个功能实现的麻烦程度，如果需要消耗的精力太大就算了，谢谢

**Comment by TheBejinning** [comment]
![image](https://user-images.githubusercontent.com/121673730/210193176-5a3908d1-672d-488c-832d-e4a4617a3993.png)
这不是有最小化功能吗

**Reply by KXST6666** [reply]
可能是我没有表达清楚，我想要那种最小化是在右下角有个小图标的那种，好像是叫做系统托盘，就是和网络声音那些图标在一块的那种，点X退到系统托盘而不是退出软件

**Comment by WForst-Breeze** [comment, marked as answer]
这个功能目前没有人反馈，可以提交一个 [**新功能提案**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)，谢谢！

**Comment by xxtg666** [comment]
有什么必要吗……PCL2开着不用会占用系统资源（差不多也有QQ的一半了），不如关掉，而且PCL2启动速度就两三秒，建议固定到任务栏方便打开

**Comment by MoYuan-CN** [comment]
进 #1162 了，去顶这条
